momento/sorted_set_increment_score.go

```diff
@@ -45,6 +48,14 @@ func (r *SortedSetIncrementScoreRequest) initGrpcRequest(client scsDataClient) e
 
 	ttlMills, refreshTTL := prepareCollectionTtl(r.CollectionTTL, client.defaultTtl)
 
+	if r.Amount == 0 {
+		return momentoerrors.NewMomentoSvcErr(
+			momentoerrors.InvalidArgumentError,
+			"Increment score amount must be passed cannot have 0 increment amount",
```

Comment:
Reference the already documented argument in the error to make it clear to the user what's wrong.

```suggestion
			"Amount must be given and cannot be 0",
```